catalog/core/catalog-core-encryption/src/main/java/org/codice/ddf/catalog/content/encryption/ContentEncryptorDecryptor.java

```diff
@@ -126,13 +134,12 @@ public InputStream decrypt(InputStream inputStream) throws EncryptorDecryptorExc
 
     byte[] iv = new byte[IV_BYTE_SIZE];
     try {
-      int bytesRead = wrappedInputStream.read(iv, 0, IV_BYTE_SIZE);
+      int bytesRead = inputStream.read(iv, 0, IV_BYTE_SIZE);
```

Comment:
✏️ There doesn't appear to be a reason to decorate the stream anymore if you aren't going to push back the last-read byte. Can delete what is now line 133.